pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -119,7 +119,7 @@ func applyWebhookPolicy(w *stablev1alpha1.WebhookPolicy, f *stablev1alpha1.Fleet
 	}
 	defer res.Body.Close() // nolint: errcheck
 	if res.StatusCode != http.StatusOK {
-		return f.Status.Replicas, false, errors.New("bad status code from the server")
+		return f.Status.Replicas, false, errors.New(fmt.Sprintf("Bad status code from the server: %s. Status: %d", urlStr, res.StatusCode))
```

Comment:
fmt.Errorf("..") instead of errors.New(fmt.Sprintf(".."))

Reply:
Thanks, updated